src/components/IconDropdown/index.js

```diff
@@ -0,0 +1 @@
+export { Dropdown, DropdownSkeleton } from 'carbon-components-react';
```

Comment:
Why move these here? I think this file should be removed. The Dropdown and DropdownSkeleton should already exported in the main index.js. Other components can import it from there (`../../index.js or whatever), or directly from carbon-components-react to prevent circular references.